lib/run.js

```diff
@@ -245,8 +245,8 @@ function run (opts, cb) {
     }
 
     if (typeof opts.duration === 'string') {
-      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration)
-      else opts.duration = Number(opts.duration.trim())
+      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration) * 1000
+      else opts.duration = Number(opts.duration.trim()) * 1000
```

Comment:
should this be `* 1000`? surely if duration was a string it should be in ms, too?